extension/js/common/org-rules.ts

```diff
@@ -161,14 +162,18 @@ export class OrgRules {
    * This is because they already have other means to obtain public keys for these domains, such as from their own internal keyserver
    */
   public canLookupThisRecipientOnAttester = (emailAddr: string): boolean => {
-    if (this.disallowLookupOnAttester()) {
-      return false;
-    }
-    const disallowedDomains = this.domainRules.disallow_attester_search_for_domains || [];
     const userDomain = Str.getDomainFromEmailAddress(emailAddr);
     if (!userDomain) {
       throw new Error(`Not a valid email ${emailAddr}`);
     }
+    // When allow_attester_search_only_for_domains is set, ignore disallow_attester_search_for_domains rule
+    if (this.domainRules.allow_attester_search_only_for_domains) {
+      return this.domainRules.allow_attester_search_only_for_domains.includes(userDomain);
+    }
+    if (this.disallowLookupOnAttester()) {
+      return false;
+    }
```

Comment:
I'll file another issue. This exists for looking up by fingerprint, which we may want to later deprecate.

Reply:
ok